.github/workflows/hil_sample_zephyr.yml

```diff
@@ -104,7 +104,7 @@ jobs:
           pip3 install -r zephyr/scripts/requirements-build-test.txt
           pip3 install -r zephyr/scripts/requirements-run-test.txt
 
-          pip3 install git+https://github.com/golioth/python-golioth-tools@v0.5.0
+          pip3 install git+https://github.com/golioth/python-golioth-tools@4e02e8fb41fc79d3d3df2a54501076bfe7578f5c
```

Comment:
This should be updated to `v0.5.1` once https://github.com/golioth/python-golioth-tools/pull/21 has merged.

Reply:
This has merged and I have updated the tag in the workflow.